src/select.rs

```diff
@@ -23,6 +23,19 @@ impl<K> Default for SelectVariant<K> {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Does it make sense to put the `BadgeVariant` into the `badge` module? In case there is one, not sure.

Reply:
here it simply change the text of the badge in the context of the select menu, i don't see how this make sense in the `badge` module. Maybe `BadgeVariant` is a poor name ?

Reply:
Hm maybe. I am just checking the patternfly docs how they name it. They seem to do it a bit differently. The "badge" is default, and only counts. You can disable it using `isCheckboxSelectionBadgeHidden` (hate that name).

And then you have "chips" instead of "badges" for the actual values, and they have an `x` to remove:

![image](https://user-images.githubusercontent.com/202474/205344151-2447454f-7a10-4967-b6fb-0e3f9f6c5ff4.png)

 I know it's a bit more work. But could you consider taking a look at this?